components/utils/Toast.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
```

Comment:
Avec la suppression en dessous import de React inutile
```suggestion
import { useEffect } from 'react';
```